django/db/migrations/serializer.py

```diff
@@ -320,6 +321,11 @@ def serialize(self):
                 return "%s.%s" % (module, self.value.__name__), {"import %s" % module}
 
 
+class UUIDSerializer(BaseSerializer):
+    def serialize(self):
+        return "uuid.UUID('%s')" % self.value, {"import uuid"}
```

Comment:
Could you use `"uuid.%s" % repr(self.value)` here instead, as in the other serializers?

    -        return "uuid.UUID('%s')" % self.value, {"import uuid"}
    +        return "uuid.%s" % repr(self.value), {"import uuid"}